README.md

```diff
@@ -114,6 +114,22 @@ It is possible to generate spdx based SBOM (software bill of materials)
 documents as part of the build. Just pass `--with-sbom` and
 it will put them into the output directory.
 
+### Cloud integration
+
+When building an image type that can be uploaded to the cloud
+(e.g. an "ami") image-builder will automatically upload if
```

Comment:
"that can be uploaded to the cloud" isn't a bad way of describing it but I'd rather be more technically accurate in this kind of documentation.  Something like:
> When building an image that's meant to work with a cloud platform ...

or

> When building an image that targets a cloud provider/platform ...

Something like that?